01-Introduction and Goals.md

```diff
@@ -4,4 +4,21 @@
 
 ## Quality Goals
 
-## Stakeholders
\ No newline at end of file
+## Stakeholders & Expectations
+
+### Wikidata Product Manager
+
+- Store structured data for Wikipedia / Wikimedia projects in a central place (Initially sitelinks and infoboxes)
+- It should be possible to create lists from the stored structured data.
+- Make the centralized data not only useful for Wikimedia projects, but also other users (researchers, apps etc.)
+- Store structured data relevant to the Wiktionary project in a meaningful way, that will also be useful for Abstract Wikipedia.
+- Store structured data relevant to files on Commons.
```

Comment:
This bullet looks a little out of place to me because everything else applies to Wikidata, but this one is Wikibase-ish functionality intended for Commons. Is this still an expectation by the Wikidata PM?

Reply:
Very true.
In reality seemingly the commons usecase and all of the architecture changes that came with it were all driven from the Wikidata / Wikimedia side of things (rather than Wikibase). But perhaps this should be moved :)

Reply:
Done in https://github.com/wmde/wikidata-wikibase-architecture/pull/6/commits/9a7728939ffceae051360bccbbc2b80d2a40fd33